### PR TITLE
feat: Switch logging to edge-core package

### LINKED ISSUE #74: Switch logging to edge-core package
Use the new common logging library that was added to edge-core.

### PR DESCRIPTION
**Pull request by DRK3**
closes #74

Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: pkg/client/edv/client_test.go [hunk 1]
```diff
@@ -60,6 +64,23 @@ func (f failingReadCloser) Close() error {
 	return fmt.Errorf("failingReadCloser always fails")
 }
 
+type TestLoggerProvider struct {
```
Comment (fqutishat) [inline pkg/client/edv/client_test.go:67]: do we need it

Comment (fqutishat) [reply pkg/client/edv/client_test.go:67]: @sudeshrshetty do we have it in aries

File: cmd/edv-rest/main.go [hunk 2]
```diff
@@ -7,13 +7,14 @@ SPDX-License-Identifier: Apache-2.0
...
 import (
-	"log"
-
 	"github.com/spf13/cobra"
+	"github.com/trustbloc/edge-core/pkg/log"
 
 	"github.com/trustbloc/edv/cmd/edv-rest/startcmd"
 )
 
+var logger = log.New("edv/cmd/edv-rest")
```
Comment (sandrask) [inline cmd/edv-rest/main.go:16]: edv-rest should be sufficient

File: pkg/client/edv/client_test.go (hunk 1 above)
Comment (sudeshrshetty) [reply pkg/client/edv/client_test.go:67]: I don't think we have it in aries,
@DRK3 why this one is being added?

Comment (DRK3) [reply pkg/client/edv/client_test.go:67]: @fqutishat Some of the unit tests I had before I could only effectively test by reading into the log - this TestLoggerProvider allows me to read into the log now with the new package. Unless there's another way? @sudeshrshetty

File: pkg/client/edv/client.go [hunk 3]
```diff
@@ -16,12 +16,14 @@ import (
...
 
+	"github.com/trustbloc/edge-core/pkg/log"
+
 	"github.com/trustbloc/edv/pkg/restapi/edv/edverrors"
 	"github.com/trustbloc/edv/pkg/restapi/edv/models"
-
-	log "github.com/sirupsen/logrus"
 )
 
+var logger = log.New("edv/pkg")
```
Comment (sandrask) [inline pkg/client/edv/client.go:25]: pkg is too broad - maybe edv-client

File: pkg/restapi/edv/operation/operations.go [hunk 4]
```diff
@@ -35,6 +35,8 @@ const (
 		docIDPathVariable + "}"
 )
 
+var logger = log.New("edv/pkg/restapi")
```
Comment (sandrask) [inline pkg/restapi/edv/operation/operations.go:38]: Usually package names logging don't have / in the  name @fqutishat @bstasyszyn @sudeshrshetty should we have some recommendations about how to name logger for package

File: cmd/edv-rest/main.go (hunk 2 above)
Comment (DRK3) [reply cmd/edv-rest/main.go]: Done

Comment (DRK3) [reply cmd/edv-rest/main.go]: I also renamed the log module in start.go to be edv-rest as well. They are both packages under the edv-rest folder, but they are separate packages. Question: is it a problem if multiple packages have the same module name? @sandrask @sudeshrshetty

File: pkg/client/edv/client.go (hunk 3 above)
Comment (DRK3) [reply pkg/client/edv/client.go]: Done

File: pkg/restapi/edv/operation/operations.go (hunk 4 above)
Comment (DRK3) [reply pkg/restapi/edv/operation/operations.go]: Updated it to just be "restapi".

**Pull request merged**